core/server/middleware/theme-handler.js

```diff
@@ -99,15 +100,14 @@ themeHandler = {
                 // Change theme
                 if (!config.get('paths').availableThemes.hasOwnProperty(activeTheme.value)) {
                     if (!res.isAdmin) {
-                        // Throw an error if the theme is not available, but not on the admin UI
-                        return errors.throwError(i18n.t('errors.middleware.themehandler.missingTheme', {theme: activeTheme.value}));
+                        return next(new errors.NotFoundError(i18n.t('errors.middleware.themehandler.missingTheme', {theme: activeTheme.value})));
```

Comment:
I think this is a 500-level error, rather than a 404? 4** is a client error, e.g. the client looking for something that doesn't exist, but this is a server error where the server is missing something vital to its operation?